iothub/device/src/TransientFaultHandling/ExponentialBackoff.cs

```diff
@@ -23,55 +23,55 @@
 namespace Microsoft.Azure.Devices.Client.TransientFaultHandling
 {
     /// <summary>
-    /// A retry strategy with backoff parameters for calculating the exponential delay between retries.
+    /// A retry strategy with back-off parameters for calculating the exponential delay between retries.
     /// </summary>
     internal class ExponentialBackoff : RetryStrategy
     {
-        private readonly int retryCount;
+        private readonly int _retryCount;
 
-        private readonly TimeSpan minBackoff;
+        private readonly TimeSpan _minBackoff;
```

Comment:
should we rename these to `_minBackOff` to camelcase the O ??

Reply:
https://en.wikipedia.org/wiki/Exponential_backoff
all lowercase

Reply:
Wikipedia is not the authority on grammar and spelling tho

Reply:
You could change the non-public variables, but I noticed there are parameters with those names and that would be a breaking change, so you might as well leave it the way it is so it is consistent.